bedrock/settings/__init__.py

```diff
@@ -28,16 +28,13 @@
 ]
 _csp_img_src = [
     "data:",
-    "mozilla.org",
     "www.googletagmanager.com",
     "www.google-analytics.com",
     "images.ctfassets.net",
```

Comment:
Hmm, aren't Contentful assets also moved to bedrock? #14198

So this could also be removed?

Reply:
Yes they should be.